package-lock.json

```diff
@@ -5241,12 +7572,12 @@
       }
     },
     "original": {
-      "version": "1.0.1",
-      "resolved": "https://forbes.jfrog.io/forbes/api/npm/npm-virtual/original/-/original-1.0.1.tgz",
-      "integrity": "sha1-sKU/9Cupl6jJzR+12q60K51pMZA=",
+      "version": "1.0.2",
+      "resolved": "https://forbes.jfrog.io/forbes/api/npm/npm-virtual/original/-/original-1.0.2.tgz",
+      "integrity": "sha1-5EKmHP/hxf0gpl8yYcJmY7MD8l8=",
       "dev": true,
       "requires": {
-        "url-parse": "1.4.1"
+        "url-parse": "1.4.3"
```

Comment:
Does this resolve the vuln? https://github.com/forbes/multiclamp/network/dependencies

Reply:
Yes it does